guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
```

Comment:
```suggestion
Use this procedure to view multi-environment hosts in Hammer CLI.
```

Reply:
Changing it to `Use this procedure to view activation keys with multiple content view environments or multi-content activation in the {ProjectWebUI} and Hammer CLI.`

Reply:
Changing it to `Use this procedure to view hosts with multiple content view environments or multi-content hosts in the {ProjectWebUI} and Hammer CLI.`  Same with activation keys.